demo/package.json

```diff
@@ -19,12 +19,12 @@
   "dependencies": {
     "nativescript-background-http": "../src",
     "nativescript-theme-core": "~1.0.4",
-    "tns-core-modules": "^5.0.0",
-    "tns-platform-declarations": "^5.0.0"
```

Comment:
tns-platform-declarations deleted. Is it expected?

Reply:
yes, it should always be devDependency not dependency